src/ui/public/timepicker/kbn_global_timepicker.js

```diff
@@ -19,7 +19,8 @@ UiModules
   return {
     template: toggleHtml,
     replace: true,
-    link: ($scope) => {
+    require: '^kbnTopNav',
```

Comment:
If you took at the corresponding [HTML template](https://github.com/elastic/kibana/blob/5e0ae4c3608e7f57e5fe18028acfaf3624ef8b69/src/ui/public/timepicker/kbn_global_timepicker.html), you'll see `kbnTopNav` written all over it. I agree that the way these all tie in together is brittle, but I think it's outside of the scope of this PR to disentangle this. At very least, this `require` call makes it more explicit that the directive relies on being nested inside the `kbnTopNav` directive.